RallySimulator.Application/Core/Vehicles/Queries/GetVehicles/GetVehiclesQuery.cs

```diff
@@ -145,14 +145,14 @@ private static string ValidateOrderBy(string orderBy)
             {
                 string columnName = orderByPart.Trim(' ').Split(' ')[0].ToLower();
 
-                if (!ValidOrderByColumnsDictionary.ContainsKey(columnName))
+                if (!ValidOrderByColumnsDictionary.TryGetValue(columnName, out string value))
                 {
                     continue;
                 }
 
                 string sortDirection = orderByPart.EndsWith(" desc", StringComparison.OrdinalIgnoreCase) ? " desc" : string.Empty;
```

Comment:
And since we named everything starting with **orderBy** we can make this `orderByDirection`?

Reply:
Done